<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QWidget" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="leftSpacing">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Entertaining Mines</string>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="tStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="mainPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>-71</y>
                <width>620</width>
                <height>580</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="FocusBarrier" name="focusBarrierTop">
                 <property name="text">
                  <string notr="true">Barrier</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="startEasy">
                 <property name="text">
                  <string>Easy Game</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Start a 9×9 game with 10 mines</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="startIntermediate">
                 <property name="text">
                  <string>Intermediate Game</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Start a 16×16 game with 40 mines</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="startDifficult">
                 <property name="text">
                  <string>Difficult Game</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Start a 30×16 game with 99 mines</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="startCustom">
                 <property name="text">
                  <string>Custom Game</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Select the board size and number of mines to play with</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="playOnlineButton">
                 <property name="text">
                  <string>Play Online</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Play with other people on the Internet</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="loadButton">
                 <property name="text">
                  <string>Load Game</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Open a game that you have saved</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="settingsButton">
                 <property name="text">
                  <string>Settings</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Go to Settings</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="infoButton">
                 <property name="text">
                  <string>Information</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Get help or view information about the game</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="exitButton">
                 <property name="text">
                  <string>Exit to Desktop</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
                 </property>
                 <property name="description">
                  <string>Leave Entertaining Mines</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="FocusBarrier" name="focusBarrierBottom">
                 <property name="text">
                  <string notr="true">Barrier</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="bottomSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="infoScreen">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Information</string>
             </property>
             <property name="margin">
              <number>9</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="FocusBarrier" name="focusBarrierInfoTop">
             <property name="text">
              <string notr="true">Barrier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="helpButton">
             <property name="text">
              <string>Help</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
             </property>
             <property name="description">
              <string>Get information about how to play</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="creditsButton">
             <property name="text">
              <string>Credits</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
             </property>
             <property name="description">
              <string>See who made the game</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="mainMenuButton">
             <property name="text">
              <string>Main Menu</string>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/icons/go-next.svg</normaloff>:/icons/go-next.svg</iconset>
             </property>
             <property name="description">
              <string>Return to the Main Menu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="FocusBarrier" name="focusBarrierInfoBottom">
             <property name="text">
              <string notr="true">Barrier</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GamepadHud" name="gamepadHud" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GamepadHud</class>
   <extends>QWidget</extends>
   <header location="global">gamepadhud.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FocusBarrier</class>
   <extends>QPushButton</extends>
   <header location="global">focusbarrier.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
